lib/apiservers/engine/backends/convert/stats.go

```diff
@@ -316,3 +419,40 @@ func (cs *ContainerStats) currentCPU() {
 	dockerCPU.CPUUsage.TotalUsage += cs.preTotalMhz
 	cs.curDockerStat.CPUStats = dockerCPU
 }
+
+const abc = "abcdefghijklmnopqrstuvxyz"
+
+// toChar will take the provided int and slice the
+// alpha constant to return the string representation.  For example
+// if 1 is provided 'a' will be returned, 2 == 'b', 3 == 'c', etc..
+func toChar(i int) string {
+	return abc[i-1 : i]
+}
```

Comment:
Yeah..it's not used...I thought originally I would need to use it...I'll remove.